README.md

```diff
@@ -70,9 +70,9 @@ exam.complete
 
 #### What are 3 ways to create a new Hash?
 
-- *give me answer*
-- *give me answer*
-- *give me answer*
+- {}
+- Hash.new
+- [].to_h #i dont know im just clutching at straws here
```

Comment:
Nice I didn't think of this, the other way to do it: https://ruby-doc.org/core-2.5.1/Hash.html#method-i-5B-5D